src/view/renderer.js

```diff
@@ -104,6 +104,13 @@ export default class Renderer {
 		 */
 		this.isFocused = false;
 
+		/**
+		 * Indicates if composition takes places inside view document.
```

Comment:
"Indicates whether text composition takes place in the document."

(you can also update `isFocused` description)